basics/Operators/Assignment.java

```diff
@@ -0,0 +1,53 @@
+package basics.Operators;
+
+public class Assignment {
+  public static void main(String[] args) {
+    int x = 5; // Assignment operator (=)
+    System.out.println("x = " + x); // Output: 5
+
+    // Addition assignment (+=)
+    x += 3;
+    System.out.println("5 += 3: " + x); // Output: 8
```

Comment:
can this changed like below to run and see the changes instead as a comment.

System.out.println("5 += 3 should returns 8 and x is " + x);